addons/getLoadoutsFromUnits/functions/fnc_createConfig.sqf

```diff
@@ -1,21 +1,17 @@
 #include "script_component.hpp"
-private _units = [];
-private _breakline = toString [13,10];
-private _structuredText = "class newFaction {";
-private _tab = "    ";
-
-{
-   if (_x isKindOf "CAManBase") then { _units pushBack _x; };
-}forEach get3DENSelected "object";
 
+private _structuredText = ["class newFaction {"];
+private _tab = "    ";
+private _units = (get3DENSelected "object") select {_x isKindOf "CAManBase"};
 
-_structuredText = _structuredText + _breakline + _tab + "class Type {";
+_structuredText pushBack (_tab + "class Type {");
 
 {
-   private _return = [_x, _breakline, _tab] call FUNC(getLoadoutAndFormat);
-   _structuredText = _structuredText + _breakline + _return;
+   private _return = [_x, _tab] call FUNC(getLoadoutAndFormat);
+   _structuredText = _structuredText + _return;
```

Comment:
what you are trying to do here is `pushBack` or `append`. I think pushback unless getLoadoutAndFormat returns array?

Reply:
Also you can use something like
_loadoutTextThingies = _units apply {[_x, _tab] call FUNC(getLoadoutAndFormat)};
_structuredText append _loadoutTextThingies